### DISCUSSION TITLE
LVGL image

### DISCUSSION THREAD

**Discussion by utya** [arendst/Tasmota#18801] [Q&A] [answered]
Hi. Guys. i tried upload png and show on dispay 9341.

sunrise = lv.img(scr)                   # create an empty image object in the current screen
sunrise.set_src("A:/Sunrise320.png")    # load "Sunrise320.png", the default drive letter is 'A:'
sunrise.move_background()   

but instead of image see, word "data".

**Comment by s-hadinger** [comment]
I'm asking the obvious question: did you upload the `Sunrise320.png` file in the file system of Tasmota ?

Can you send the logs and a picture?

**Reply by sfromis** [reply]
With a simple test, I got that image displayed on an ILI9341. Note that for an image of that size to work, you must have a board with PSRAM available. Beware that if your image handling of binary data failed at some point, you might have broken it. Using the image from the docs, I see an image file size of 112062 bytes.

**Reply by sfromis** [reply]
If the file size is causing trouble, here's a 128x128 test image with a file size of only 655 bytes.
![tasmota-logo-rgb](https://github.com/arendst/Tasmota/assets/47082390/2a9f747f-189a-47a7-b145-08c77644b363)

**Reply by utya** [reply]
i tried sunrise from sample, and 
![ok-16](https://github.com/arendst/Tasmota/assets/11736439/940dbe6a-239a-41aa-bb0c-bf23dd1fbf67)
I don't have psram
https://grobotronics.com/esp32-development-board-devkit-v1.html?sl=en

with your image it works. but now i have new issue, after 1 min the screen begin white, i think problem in wiring.

**Reply by s-hadinger** [reply, marked as answer]
If you don't have PSRAM, forget about images. They just don't fit in the on-board memory.
